packages/server/package.json

```diff
@@ -112,6 +116,7 @@
     "p-queue": "6.1.0",
     "parse-domain": "2.0.0",
     "pluralize": "8.0.0",
+    "promise-retry": "^1.1.1",
```

Comment:
i think we ended up writing our own thing to do this, and its either no longer necessary or we need to use the one we wrote in `packages/network`